src/test/java/ru/netology/domain/RadioTest.java

```diff
@@ -7,151 +7,86 @@
+    public void setCurrentStationAndVolume() {
+        Radio radio = new Radio(5, 5, 20);
         assertEquals(5, radio.getCurrentStation());
+        assertEquals(5, radio.getCurrentVolume());
+        assertEquals(20, radio.getMaxStation());
 
     }
 
     @Test
     public void setCurrentStationMax() {
-        Radio radio = new Radio();
-        assertEquals(0, radio.getCurrentStation());
-        radio.setMaxStation(9);
-        radio.setMinStation(0);
-        radio.setCurrentStation(10);
+        Radio radio = new Radio(22, 50, 20);
         assertEquals(0, radio.getCurrentStation());
-
     }
 
     @Test
     public void setCurrentStationMin() {
-        Radio radio = new Radio();
-        assertEquals(0, radio.getCurrentStation());
-        radio.setMaxStation(9);
-        radio.setMinStation(0);
-        radio.setCurrentStation(-1);
+        Radio radio = new Radio(-1, 50, 20);
         assertEquals(0, radio.getCurrentStation());
 
     }
 
-
-
-    @Test
-    public void setCurrentVolume() {
-        Radio radio = new Radio();
-        assertEquals(0, radio.getCurrentVolume());
-        radio.setMaxVolume(10);
-        radio.setMinVolume(0);
-        radio.setCurrentVolume(5);
-        assertEquals(5, radio.getCurrentVolume());
-
-    }
-
     @Test
     public void setCurrentVolumeMax() {
-        Radio radio = new Radio();
-        assertEquals(0, radio.getCurrentVolume());
-        radio.setMaxVolume(10);
-        radio.setMinVolume(0);
-        radio.setCurrentVolume(11);
+        Radio radio = new Radio(5, 101, 20);
         assertEquals(0, radio.getCurrentVolume());
 
     }
 
     @Test
     public void setCurrentVolumeMin() {
-        Radio radio = new Radio();
-        assertEquals(0, radio.getCurrentVolume());
-        radio.setMaxVolume(10);
-        radio.setMinVolume(0);
-        radio.setCurrentVolume(-1);
+        Radio radio = new Radio(5, -1, 20);
         assertEquals(0, radio.getCurrentVolume());
 
     }
 
     @Test
     public void nextStation() {
-        Radio radio = new Radio();
-        radio.setMinStation(0);
-        radio.setMaxStation(9);
-        radio.setCurrentStation(5);
-        radio.NextStation();
+        Radio radio = new Radio(5, 50, 30);
+        radio.nextStation();
         assertEquals(6, radio.getCurrentStation());
     }
 
     @Test
     public void nextStationMax() {
-        Radio radio = new Radio();
-        radio.setMinStation(0);
-        radio.setMaxStation(9);
-        radio.setCurrentStation(9);
-        radio.NextStation();
+        Radio radio = new Radio(20, 50, 20 );
+        radio.nextStation();
         assertEquals(0, radio.getCurrentStation());
     }
 
     @Test
-    public void nextVolume() {
-        Radio radio = new Radio();
-        radio.setMinVolume(0);
-        radio.setMaxVolume(10);
-        radio.setCurrentVolume(5);
-        radio.NextVolume();
-        assertEquals(6, radio.getCurrentVolume());
+    public void NextVolume() {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.